### ISSUE TITLE
Pull in Latest Data from MDN

### ISSUE THREAD

**Issue by mahirshah** [ben-eb/css-values#29]
The data.json has not been updated in a while. Would be good to acquire the latest data from MDN.

**Comment by ben-eb** [comment]
Yeah, actually MDN have moved the data to https://github.com/mdn/data. I'm hoping that they will start publishing it to npm so that it's easier to consume, but for now we might have to load it from github instead.

Much of https://github.com/ben-eb/css-values/blob/master/src/data.js needs rewriting as the data has undergone numerous improvements; it's no-longer HTML encoded also.
